### PR TITLE
Rebranding dev branch to 4.3.0-rtm

### PR DESCRIPTION
**Pull request by alpaix**
//cc @rrelyea

### CODE REVIEW

File: build/config.props [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <SemanticVersion Condition=" '$(SemanticVersion)' == '' ">4.3.0</SemanticVersion>
     <!-- We need to update this netcoreassembly build number with every milestone to workaround any breaking api
     changes we might have made-->
-    <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">4</NetCoreAssemblyBuildNumber>
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview$(NetCoreAssemblyBuildNumber)</ReleaseLabel>
+    <NetCoreAssemblyBuildNumber Condition=" '$(NetCoreAssemblyBuildNumber)' == '' ">5</NetCoreAssemblyBuildNumber>
```
Comment (nkolev92) [inline build/config.props:8]: Isn't Preview3 with BuildNumber 5 or something like that?

Comment (nkolev92) [reply build/config.props:8]: Actually...Preview 3 and Preview 4 have the same Build Number...4....not sure how this was supposed to be.

Comment (rohit21agrawal) [reply build/config.props:8]: @nkolev92 that was intentional. @zhili1208 made that change when he was on hotseat.

Comment (alpaix) [reply build/config.props:8]: `NetCoreAssemblyBuildNumber` should be incremented with every minor release. It was a coincidence when it was equal `previewN` label.

**Review by emgarten — APPROVED** [review]

**Comment by alpaix** [comment]
🔔 @rohit21agrawal Could you please take a look and approve if everything is okay?

**Review by rohit21agrawal — APPROVED** [review]

**Pull request merged**